agent.go

```diff
@@ -52,10 +57,18 @@ type Payload struct {
 	HostDate       time.Time           `json:"hostdate"`
 	LookupHost     map[string][]string `json:"nslookup"`
 	IPs            map[string][]string `json:"ips"`
+	NetworkProbes  []ProbeResult       `json:"network_probes"`
 	ZeroExtender   []int8              `json:"zero_extender"`
 }
 
-func sendInfo(srvEndpoint, podName string, nodeName string,
+// ProbeResult structure for network probing results
+type ProbeResult struct {
+	EndPoint        string
+	Total           int
+	ContentTransfer int
```

Comment:
Please also add the following:
TCPConnection
DNSLookup
Connect
ServerProcessing